### DISCUSSION TITLE
Migrating an existing project that uses Docker

### DISCUSSION THREAD

**Discussion by nprimmer** [noop-inc/documentation#34] [Questions] [answered]
If I have a project that's already using Docker...
- How would I migrate the existing Dockerfile or docker-compose.yml into a Noop app.yml?
- Does any tooling exist to perform the conversion for me?
- Does Noop support the use of multi-stage builds in the same manner as Docker?

**Comment by milin82** [comment, marked as answer]
- We don't have a conversion tool available just yet but we plan to provide a tool soon (this quarter or next) to help convert docker-compose files to Noop blueprints
- Yes, Noop supports multi-stage builds
